packages/backend-api/install/install.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+
+INSTALL_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+PROJECT_BASE_PATH=`[ -d "$PWD/project-base" ] && echo "$PWD/project-base" || echo "$PWD"`
```

Comment:
Thanks for the change. I'm just wondering, if you already have proper path to project-base, why you forbid the installation from a different path?
If you use 
```
PROJECT_BASE_PATH=`realpath -e ${INSTALL_DIR}/../../../project-base`
``` 
(notice `-e`) command fails when path does not exists and otherwise everything else working properly even when I run it from different folder

Reply:
two reasons
* `-e` doesn't allow me show human error message, it fails on "it doesn't exists"
* for the monorepo this approach is valid, but for project-base, there is just `../../../..` and that would exist